### DISCUSSION TITLE
求助，宝塔环境下用Docker部署后如何开启SSL？

### DISCUSSION THREAD

**Discussion by CodeFishK** [GaiZhenbiao/ChuanhuChatGPT#520] [Q&A]
**请教宝塔环境下用Docker部署后如何开启SSL？**  

本人用的公网服务器Docker部署，部署后开启端口反代并开启SSL，用IP加端口可以正常使用；  

**用域名访问时会有如下报错：**
<img width="1719" alt="image" src="https://user-images.githubusercontent.com/38446347/229394729-12131ffd-6140-48a8-88c0-a5d7fd456c8a.png">
**附上网站相关配置截图：**
<img width="803" alt="image" src="https://user-images.githubusercontent.com/38446347/229394980-a2f19392-754d-4d10-a07f-7c410484e698.png">
<img width="815" alt="image" src="https://user-images.githubusercontent.com/38446347/229395054-4ea77419-ab75-45ab-88c9-9cf6a93244fc.png">

**Comment by LaoJiuYes** [comment]
我也是，你解决了吗？
